Cargo.toml

```diff
@@ -34,3 +34,4 @@ colored = "2"
 
 [dev-dependencies]
 quickcheck = "0.9.2"
+trycmd = "0.13.4"
```

Comment:
seems odd that it creates empty files for empty stderr, maybe patch that upstream?

Reply:
The documentation states that if the files are not present the output is not checked. tested and it behaves like that. I don't think those empty files are an issue

Reply:
Ah... unfortunate difference to rustc's ui tests. 🤷